types/ansi-align/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for ansi-align 3.0
+// Project: https://github.com/nexdrew/ansi-align
+// Definitions by: Leon Si <https://github.com/leondreamed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Align = 'left' | 'center' | 'right';
+
+interface AnsiAlign {
+    (text: string, opts?: { align?: Align; split?: string; pad?: string }): string;
+    (text: ReadonlyArray<string>, opts?: { align?: Align; pad?: string }): string[];
```

Comment:
why this limitation? There is no need to discriminate by missing/existing 'split' option (seems it's always used in native JS setup and has no branching logic role)

Reply:
I think if the text is passed as an array, the "split" option is never used (which the documentation also says here):
![image](https://user-images.githubusercontent.com/36966635/207157432-751d9ac6-17cf-41a9-9cae-cfb38d02d884.png)

I wasn't sure if I was supposed to add this logic to the types, but let me know if I should unify the calls!